### PR TITLE
Adds armor accessories and armor rebalance

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request

Adds three types of armor accessories (pin it to your uniform to buff your melee, ballstic or laser defence) which are available as a Syndicate version for 2 TC each and are invisible on examine, as well as a Nanotrasen version which are not invisible but require advanced research. 

Adds the tactical turtleneck to the uplink, which is a cheap if obvious armored uniform if you don't want to buy a full chameleon kit. Reduces the cost of Syndicate space suit to 2 TC (from 3) and tripled energy combat shield integrity. 

## Why It's Good For The Game

Antags will often want or need to face Security head-on, so having additional armor options helps with making that strategy viable. These accessories also give some much needed late-game progression to Security which the department is sorely lacking. 

Tactical turtleneck and Syndicate space suit are some inexpensive armored options if you don't mind being an obvious Syndie. Energy combat shield got heavily nerfed during the whole shield rework, hopefully this change will make it viable again. 

## Changelog
:cl:
add: Armor accessories now available as an invisible version for 2 TC or visible version late in the research tree. 
add: Tactical turtleneck now available for 1 TC.
balance: Syndicate space suit now costs 2 TC. 
balance: Energy combat shield integrity has been tripled. 
/:cl:

Ports: 
[#9149](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9149)

### CODE REVIEW

**Comment by bloons3** [comment]
This removes the need for security to pick between laser/ballistic protection, correct?
They get it all now.

**Comment by VictorVow** [comment]
Reflector vest and bulletproof vest are not the only two options. Additionally there is riot armour and normal vests, each with differing armour values.  The values for the specialised armours tend to be about ~60 for the targeted damage type and ~10 for the non-targeted types (eg bulletproof armour has 60 bullet but 10 laser/energy and 15 melee).

Pairing that with ablative padding (laser protection accessory) would give you 60 bullet, 30 laser, 20 energy and 15 melee so it's not quite up to normal armour vest levels (which is 30-40 armour for all types) if you were indeed to seeking to be average all-round. Also a bulletproof vest here would only provide that level of defence on the chest, whereas other armour like riot armour covers the entire body, however at a loss of speed. All in all there is still a great number of tradeoffs to be made in selecting what combination of armour and accessory best suits the challenges faced in any given shift.

**Comment by Isy232** [comment]
Why did you buff shields?

**Comment by VictorVow** [comment]
Just the energy combat shield, and like I said in OP they're crap currently. Nukies never get into melee unless they're doing an energy chainsaw build so the shield melee block is irrelevant, it's prior purpose for blocking all energy/laser projectiles and some ballistic is woeful right now as it only blocks about three lasers before turning off.

**Comment by bloons3** [comment]
Sec gets 1 type of armor for each situation because they have to compromise.

Basic armor isn't great at anything, but protects against most threats.

Traitor stole lasers, go laser armor.
Nuke ops?  Go ballistic armor, and grab lasers because then you can shoot them through walls.
Revs? Go riot armor, because they'll likely have melee weapons.

This is armor powercreep.

**Comment by bloons3** [comment]
>Ports:
From Citadel



Do you link this or credit the original PR anywhere?

**Comment by VictorVow** [comment]
The numbers I gave examples for show that even with an accessory the armour is still not generalised. "Powercreep" can be referred to anything which gives a new late-game feature, and Security needs something minor like this if Robotics are getting Durands, Botany making eswords and hundreds of grenades, Xenobio and Genetics making themselves gods, etc. An accessory as a late-game item as nothing in comparison. 

Crediting the original codebase is sufficient but I've updated the link to the actual PR.

**Comment by bloons3** [comment]
>Security needs something minor like this

Security already holds control over all guns printed on the station by their control of mindshields and firing pins.
Security also hold control over all guns **ordered** on the station by virtue of keeping a close eye on Cargonia.

Security controls the late game weapon area because you can't get working guns without Sec's cooperation, because any printed gun needs a firing pin.
You can't even arm mechs unless Sec works with robotics and gives them gun modules and ammo from the Security Lathe.

**Comment by VictorVow** [comment]
Indeed they do control most guns. However you can easily murderbone with weapons from any of the aforementioned jobs, but not with what is likely to be a maximum two energy guns in the future. Late game Security is just not as strong as any other department. 

> Security also hold control over all guns ordered on the station by virtue of keeping a close eye on Cargonia.

Gun crates can be ordered and arrive within 1 minute, the auto rifle crates taken into space and PKA'd open, then guns put into a backpack whilst meanwhile the shuttle returned all before the department officer checks the console again. Nobody can check it every minute and most of the time when guns are actually discovered it's because the AI happened to see it. 

Robotics specifically got nerfed very recently however a Durand can break into the Brig easily to arm itself with lethals, and coders have already discussed making the mech fab emaggable to make lethals more easily obtainable since currently mechs for antags are underwhelming. 

Anyway this is veering off topic. Is Security armour overpowered? No, funny antag ballistics/esword/cult sword/whatever still butchers with ease. The difference is now if you wait until late-game to try to murder Security they actually will be harder to kill for once.

**Comment by bloons3** [comment]
> Late game Security is just not as strong as any other department.

Late Game Security has a smorgasbord of weapons to pick from. The lowly shotgun has a spread of upgraded shotgun shells, some of which even mimic pulse rifles. The most powerful shotgun shell replaces the pellets with lasers.

Security can also select from the X-Ray gun, the pistol sized disabler/ion gun, the Tesla revolver.

Security is not starved for late-game progression.

> but not with what is likely to be a maximum two energy guns in the future

Ah yes, a grudge PR for something that was test merged without the greytide rising up and killing every Sec officer they could find.

**Comment by VictorVow** [comment]
See the shotgun nerf PR, it's about to become irrelevant. X-ray gun is very niche since it requires X-ray vision to have any benefit over energy guns, ion rifle will also become bulky and Tesla revolver harms the user. I personally nerfed stunbatons because I agreed in that specific instance but the nerfs have continued, the most recent target being flashbangs. 

Yes it's some added variety but you're here trying to argue these are comparable to the early to late game progression that other departments undergo and that no other additional items should be ever added to this department. When Botany, Mining, Xenobio etc are far more overpowered it's rather odd you're taking such a strong stance against this particular job. 

> Ah yes, a grudge PR for something that was test merged without the greytide rising up and killing every Sec officer they could find.

Did anybody ever insinuate this would be the case?

**Comment by bloons3** [comment]
You mentioned that a reason for this PR was another as-of-yet unmerged PR that modifies the size of energy guns.

In response to that PR, you made this PR to buff Security, because (as you've said before) that other PR would nerf Security.

**Comment by VictorVow** [comment]
This PR was actually made after a separate discussion with Bacon about how disablers should stay strong against greytide but antags should have more defensive options. Giving this item to Security additionally was in my mind unrelated to the gun nerf (because having more armour isn't equivalent to being able to carry more guns), instead because I just generally think Sec lacks late-game progression.

File: code/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -378,3 +378,37 @@
...
+/////////////////////
+//Armour Accessories//
+/////////////////////
+
+/obj/item/clothing/accessory/softpadding
+	name = "soft padding"
+	desc = "A protective padding meant to cushion the wearer from melee harm."
+	icon_state = "padding"
+	item_color = "nothing"
+	armor = list("melee" = 20, "bullet" = 10, "laser" = 0, "energy" = 0, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = -20, "acid" = 45)
```
Comment (PowerfulBacon) [inline code/modules/clothing/under/accessories.dm:391]: HOS jumpsuit: +10
Soft padding: +20 (30)
Ash Drake Armour: +70 (100)
You can get 100 melee armour on the body and limbs

Comment (PowerfulBacon) [reply code/modules/clothing/under/accessories.dm:391]: All you need is any armour with +70 melee resistance

**Comment by PowerfulBacon** [comment]
Id be all 👍 on this PR except sec can make them too

File: code/modules/clothing/under/accessories.dm (hunk 1 above)
Comment (VictorVow) [reply code/modules/clothing/under/accessories.dm:391]: I'd argue this is more an issue with Lavaland's insane loot but to be fair I was just using the original PR's values, I can halve them.

**Comment by VictorVow** [comment]
It's late-game tech, Sec doesn't get *any* armour upgrades over the course of a round, Sec doesn't have much tech progression, the trend of PRs lately is widespread Sec nerfs and antag buffs, I'm halving these accessories effectiveness, etc. Sec having access to these as well comparatively is not a big deal.

**Comment by bloons3** [comment]
Justifying a buff by pointing to another buff and saying that the new buff will counter the old buff is how you get powercreep.

**Comment by PowerfulBacon** [comment]
> 
> 
> Justifying a buff by pointing to another buff and saying that the new buff will counter the old buff is how you get powercreep.

Yea, and sec has already had quite a lot of powercreep

**Comment by VictorVow** [comment]
I'm quite confused by that statement when a  year or so ago batons were one stun and done, and a little before that tasers were the same thing but ranged. And it is equally baffling when most sec weaponry is being nerfed and has been recently. 

Plus on top of that counters like flashproofing, shields and bang protection are more common than ever. I must truly be missing a lot if after all of this there can still be considered to be any leftover "powercreep".

**Comment by aguyiguess** [comment]
>sec has already had quite a lot of powercreep

bro do you have stage 4 alzheimer's like have you forgotten the past year or two

**Comment by PowerfulBacon** [comment]
Yea valid point if everything else wasn't continuously nerfed too to make combat longer than 1 click overall

**Comment by VictorVow** [comment]
Agreed, which is why I think nerfs are more the trend rather than powercreep across both antags and sec.

**Comment by PowerfulBacon** [comment]
actually ill let the vote go on a bit longer, but it looks like its mostly failing due to them being printable.

**Comment by utilbird** [comment]
Closing due to failed vote and maintainer verdict

**Pull request closed without merging**